### ISSUE TITLE
Where are the Python communities and enthusiasts?

### ISSUE THREAD

**Issue by kytrinyx** [exercism/python#1397]
As we move towards the launch of the new version of Exercism we are going to be ramping up on actively recruiting people to help provide feedback.

Our goal is to get to 100%: everyone who submits a solution and wants feedback should get feedback. Good feedback. You can read more about this aspect of the new site here: http://mentoring.exercism.io/

To do this, we're going to need a lot more information about where we can find language enthusiasts.

* Is Python supported by one or more large organizations?
* Does Python have an official community manager?
* Do you know of specific communities (online or offline) that are enthusiastic about Python? (Chat communities, forums, meetups, student clubs, etc)
* Are there popular conferences for Python? (If so, what are some examples?)
* Are there any organizations who are targeted specifically at getting certain subgroups or demographics interested in Python? (e.g. kids, teenagers, career changers, people belonging to various groups that are typically underrepresented in tech?)
* Are there specific groups or programs dedicated to mentoring people in Python?
* Are there popular newsletters for Python?
* Is Python taught at programming bootcamps? (If so, what are some examples?)
* Is Python taught at universities? (If so, what are some examples?)

In other words: where do people care a lot and/or know a lot about Python?

This is part of the project being tracked in https://github.com/exercism/meta/issues/103

**Comment by cmondorf** [comment]
* Is Python supported by one or more large organizations?

That would be the [Python Software Foundation](https://www.python.org/psf/).

* Do you know of specific communities (online or offline) that are enthusiastic about Python? (Chat communities, forums, meetups, student clubs, etc)?

There's a pretty active [slack channel](pythondev.slack.com) and [Reddit](https://www.reddit.com/r/Python/)

**Comment by kytrinyx** [comment]
Thanks @cmondorf!

**Comment by dmerejkowsky** [comment]
> Are there popular conferences for Python? (If so, what are some examples?)

PyCon comes to mind.

> Are there popular newsletters for Python?

I'm subscribed to the [import Python newsletter](http://importpython.com/newsletter/)

**Comment by lbillinghamtn** [comment]
[Pycon North America](us.pycon.org) and [EuroPython](europython.eu) are the 2 biggest conferences (I think).

[DjangoGirls](https://djangogirls.org) and [PyLadies](http://www.pyladies.com) aim to increase the number of women using python.

Micropython along with the BBC microbit are aimed at getting kids into coding (by way of python)

**Comment by willingc** [comment]
@kytrinyx Reaching out to the regional Python conferences would be a good way to recruit folks too.

**Comment by kytrinyx** [comment]
Thank you everyone! ❤️ 💛 💚 💙 💜 🖤 
I've got what I need for now; closing this out. 🌼
